### PR TITLE
Add Quefrency Rev. 3

### PR DESCRIPTION
**Pull request by nooges**
## Description

Add Quefrency Rev. 3

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/quefrency/rev3/rev3.c [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#include "quefrency.h"
```
Comment (drashna) [inline keyboards/keebio/quefrency/rev3/rev3.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/keebio/quefrency/rev3/rev3.h [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+#include "quefrency.h"
+#include "quantum.h"
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+    #include <avr/io.h>
+    #include <avr/interrupt.h>
+#endif
+#endif
```
Comment (drashna) [inline keyboards/keebio/quefrency/rev3/rev3.h:12]: Pretty sure this block isn't needed.
```suggestion
```

File: keyboards/keebio/quefrency/rev3/rev3.h [hunk 3]
```diff
@@ -0,0 +1,194 @@
+#pragma once
```
Comment (drashna) [inline keyboards/keebio/quefrency/rev3/rev3.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/keebio/quefrency/rev3/config.h [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xCB10
+#define PRODUCT_ID      0x3257
+#define DEVICE_VER      0x0300
+#define MANUFACTURER    Keebio
+#define PRODUCT         Quefrency Rev. 3
+#define DESCRIPTION     Split 60/65 percent staggered keyboard
```
Comment (fauxpark) [inline keyboards/keebio/quefrency/rev3/config.h:27]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**